e2e/pages/adf/process_cloud/taskFiltersCloudComponent.ts

```diff
@@ -49,6 +49,21 @@ export class TaskFiltersCloudComponent {
         return this.filter.click();
     }
 
+    clickCustomFilters() {
```

Comment:
let's move these methods from taskFiltersCloudComponent.ts to tasksCloudDemoPage.ts, cause these dropdowns  are part of demo-shell only.